concourse/pipelines/deploy-cloudfoundry.yml

```diff
@@ -347,6 +356,9 @@ jobs:
     serial: true
     plan:
       - aggregate:
+        - get: pipeline-trigger
+          passed: [ 'generate-manifest' ]
+          trigger: true
```

Comment:
Yes, we think that it should be triggered by both. It will only run if generate-manifest job has passed and it looks like logical consequence of manifest generation.